src/spaceapi_client.rs

```diff
@@ -14,20 +14,22 @@ use std::io;
 pub struct SpaceApiClient {
   last_fetch: i64,
   webcams: Vec<String>,
+  location: Option<Location>,
```

Comment:
Ah, thanks for the explanation :)

If I understand the code correctly, you're using the `webcams` and `location` fields for caching, right? Maybe it would be easier to cache the entire response instead of replicating the structure?

```rust
pub struct SpaceApiClient {
    last_fetch: chrono::DateTime<UTC>,
    response: spaceapi::Status,
}
```